lensterxyz/lenster
# fix: zustand warning about deprecated fields

## What does this PR do?

Fixes #2155

## Type of change


## How should this be tested?

Verify loading/storing of unsent messages (lenster.xyz/messages).
Verify that the badge (red dot) for new messages works.
